README.md

```diff
@@ -56,7 +56,7 @@ For any CSM Operator and driver issues, questions or feedback, join the [Dell Te
 ## Supported Platforms
 Dell Container Storage Modules Operator has been tested and qualified with 
 
-    * Upstream Kubernetes cluster v1.24, v1.25, v1.26
+    * Upstream Kubernetes cluster v1.21, v1.24, v1.25, v1.26
```

Comment:
We dont have to call out the support for K8s v1.21. So please revert the change in this file